plugins/node/opentelemetry-instrumentation-cassandra/src/types.ts

```diff
@@ -15,6 +15,12 @@
  */
 
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import { Span } from '@opentelemetry/api';
+import type * as CassandraDriver from 'cassandra-driver';
+
+export interface CassandraDriverResponseCustomAttributeFunction {
+  (span: Span, response: CassandraDriver.types.ResultSet): void;
```

Comment:
We usually construct a hook signature to have `span` object and then an `Info` object which contains all the relevant data for this hook.
In the future, more data could be added to the hook (for example: moduleVersion) and then we end up with long signatures with many arguments which are harder to work with.

You can see an example [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/b35277bb5fc66910e8942bc0b64347b68ecffa26/plugins/node/instrumentation-amqplib/src/types.ts#L20)

Reply:
True, I'll construct an object which will contain the response for now, which will allow for easier data addition in the future